pkg/gits/git_cli.go

```diff
@@ -220,7 +220,12 @@ func (g *GitCLI) RemoteUpdate(dir string) error {
 
 // Stash stashes the current changes from the given directory
 func (g *GitCLI) Stash(dir string) error {
-	return g.gitCmd(dir, "stash")
+	return g.gitCmd(dir, "stash", "push")
```

Comment:
Rename to `StashPush` for symmetry and to make things more explicit when seeing this func called.